node-runtime/spec/rest/rest.spec.ts

```diff
@@ -0,0 +1,223 @@
+import { Parser } from "@sdkgen/parser";
+import { generateNodeClientSource, generateNodeServerSource } from "@sdkgen/typescript-generator";
+import axios from "axios";
+import { unlinkSync, writeFileSync } from "fs";
+import { Context, SdkgenHttpServer } from "../../src";
+
+const ast = new Parser(`${__dirname}/api.sdkgen`).parse();
+
+writeFileSync(`${__dirname}/api.ts`, generateNodeServerSource(ast, {}).replace("@sdkgen/node-runtime", "../../src"));
+const { api } = require(`${__dirname}/api.ts`);
+unlinkSync(`${__dirname}/api.ts`);
```

Comment:
[Since TypeScript 2.4 dynamic imports are available](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-4.html#dynamic-import-expressions), which allows importing files from [data URIs](https://en.wikipedia.org/wiki/Data_URI_scheme). This approach should be better than using I/O, but it's just a suggestion.

Reply:
ts-jest doesn't seems to handle it well. I'll give another try later, but anyway I think this can be changed in a new PR.